src/fontra/core/classes.py

```diff
@@ -85,13 +85,15 @@ class FontSource:
     name: str
     location: Location = field(default_factory=dict)
     verticalMetrics: dict[str, FontMetric] = field(default_factory=dict)
+    italicAngle: float = 0
     guidelines: list[Guideline] = field(default_factory=list)
     customData: CustomData = field(default_factory=dict)
 
 
 @dataclass(kw_only=True)
 class FontMetric:
     value: float
+    zone: int = 0
```

Comment:
This should be a float, like all other metric numbers.

Reply:
thanks.